oauth2/smart/patient-select.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/**
+ * patient-select.php
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <stephen@nielson.org>
+ * @copyright Copyright (c) 2021 Stephen Nielson <stephen@nielson.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Session\SessionUtil;
+use OpenEMR\Core\Header;
+
+if ($oauthLogin !== true) {
+    $message = xlt("Error. Not authorized");
+    SessionUtil::oauthSessionCookieDestroy();
+    echo $message;
+    exit();
+}
+
+// make sure we have our patients set
+$errorMessage = $errorMessage ?? "";
+$patients = $patients ?? [];
+$redirect = $redirect ?? "";
+$searchAction = $searchAction ?? "";
+$fname = $searchParams['fname'] ?? "";
+$mname = $searchParams['mname'] ?? "";
+$lname = $searchParams['lname'] ?? "";
+$hasMore = $hasMore ?? false;
+
+?>
+<html>
+<head>
+    <title><?php echo xlt("OpenEMR Authorization"); ?></title>
+    <?php Header::setupHeader('opener'); ?>
+    <script>
+        (function(window) {
+            window.opener = null;
+        })(window || {});
+        <?php require($GLOBALS['srcdir'] . "/restoreSession.php"); ?>
```

Comment:
this seems odd. was this needed for something?

Reply:
no, originally this was in a dialog.  Will remove.